python/ray/tests/test_runtime_env.py

```diff
@@ -13,20 +14,27 @@
 from ray._private.utils import get_conda_env_dir, get_conda_bin_executable
 from ray.job_config import JobConfig
 
+PKG_DIR = Path(tempfile.gettempdir()) / "ray" / "session_latest" \
+  / "runtime_resources"
+
 driver_script = """
 import sys
 import logging
 sys.path.insert(0, "{working_dir}")
 import test_module
 import ray
-
+import ray.util
+import os
 job_config = ray.job_config.JobConfig(
     runtime_env={runtime_env}
 )
 
-ray.init(address="{redis_address}",
-         job_config=job_config,
-         logging_level=logging.DEBUG)
+if os.environ.get("CLIENT"):
```

Comment:
```suggestion
if os.environ.get("USE_RAY_CLIENT"):
```

Consider for clarity.